numba_cuda/numba/cuda/tests/cudapy/test_dispatcher.py

```diff
@@ -1,7 +1,7 @@
 # SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 # SPDX-License-Identifier: BSD-2-Clause
 
-from numba.cuda.cudadrv.driver import CudaAPIError
+from cuda.core.experimental._utils.cuda_utils import CUDAError
```

Comment:
Is this something that we need to better expose publicly in `cuda.core`?

Reply:
Yes, but I don't think that should block this PR.